sktime/datasets/_readers_writers/arff.py

```diff
@@ -13,9 +13,9 @@
 from sktime.datasets._readers_writers.utils import get_path
 from sktime.transformations.base import BaseTransformer
 
-# ==================================================================================================
+# ================================================================================================== # noqa
```

Comment:
Please don't use a broad `noqa`.

Reply:
Also, you can possibly reduce as you did below.

Reply:
yeah I have to explicitily mention the exception rule